documentation/book/oauth/proc-oauth-server-config.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following module:
+//
+// con-oauth-config.adoc
+
+[id='proc-oauth-server-config-{context}']
+= Configuring an {oauth} authorization server
+
+This procedure describes how to configure an authorization server for integration with {ProductName}.
+
+Depending on the chosen xref:con-oauth-authentication-client-options-{context}[authentication flow], you can use the authorization server to:
+
+* Configure authentication for Kafka brokers
+* Configure and authorize clients
+* Configure users and roles
+* Obtain access and refresh tokens
+
+User credentials are managed by an authorization server using a _realm_, which describes security policies.
+The _realm_ is the authentication provider for clients.
+
+To use {oauth} with {ProductName}, you need a deployment of an authorization server to be able to create and manage authentication realms.
+
+NOTE: If you already have an authorization server deployed, you can skip the deployment step and use your current deployment.
+
+.Prerequisites
+
+* {ProductName} and Kafka is running
+
+For an authorization server deployment:
+
+* Check the supported configuration for the authorization server
+* Installation requires a user with a cluster-admin role, such as system:admin
+
+.Procedure
+
+. Deploy the authorization server to your cluster.
+. Access the CLI or admin console for the authorization server to create the {oauth} policies for {ProductName}.
++
+Consult the product documentation for the authorization server for information on creating policies.
+
+. Create and enable a _realm_.
```

Comment:
@mstruk Is _realm_ a generic term in OAuth world? Would it be understandable to non-Keycloak users?

Reply:
I don't see any reference in the The OAuth 2.0 Authorization Framework, but there are lots of definitions around in relation to HTTP and security.  'A realm is a security policy domain defined for a web or application server. A realm contains a collection of users, who may or may not be assigned to a group.' 

We say this: 'User credentials are managed by an authorization server using a _realm_, which describes security policies. The _realm_ is the authentication provider for registered clients.'

@mstruk - Is this step optional according to how the authorization server manages policy?

Reply:
@scholzj No, realm is not a recognised term in OAuth world, it might be in identity provider world? It's something Keycloak uses to basically provide multitenancy - a whole separate space of users, clients, roles and other configuration.
@PaulRMellor It is optional. You always use some realm, and by default the realm is called 'master' and exists OOTB. However, if you want to control things like max session timeout, and if you have web based applications coexisting within the same realm, and if you want to to configure clients with long-lived access tokens,  then you may have some issues, and complexities how to properly configure that. Having a separate realm makes it easier, and it may be easier to control which users / clients have access to your cluster.

Reply:
It seems as if we want to give users details on how to configure an unspecified authorization server, but we can only talk about details specifically for Keycloak or Hydra. If we want to stay generic we can only give general steps.

I would remove L37-L40.